packages/config/src/config/moonbase/assets/eq.ts

```diff
@@ -0,0 +1,41 @@
+import { AssetSymbol, ChainKey } from '../../../constants';
+import {
+  EqBalancesFee,
+  EqBalancesSuccessEvent,
+} from '../../../extrinsic/eqBalances';
+import {
+  assets,
+  balance,
+  chains,
+  extrinsic,
+  withdraw,
+} from '../moonbase.common';
+import { MoonbaseXcmConfig } from '../moonbase.interfaces';
+
+const asset = assets[AssetSymbol.EQ];
+const eqa = chains[ChainKey.EquilibriumAlphanet];
+const eqId = 25_969;
+
+export const EQ: MoonbaseXcmConfig = {
+  asset,
+  origin: eqa,
+  deposit: {
+    [eqa.key]: {
+      source: eqa,
+      balance: balance.systemEquilibrium(eqId),
+      extrinsic: extrinsic
+        .eqBalances()
+        .xcmTransfer()
+        .successEvent(EqBalancesSuccessEvent.ExtrinsicSuccess)
+        .asset(eqId)
+        .fee(EqBalancesFee.ThisAccWillPay),
+    },
+  },
+  withdraw: {
+    [eqa.key]: withdraw.xTokens({
+      balance: balance.systemEquilibrium(eqId),
+      destination: eqa,
+      feePerWeight: 0.1,
```

Comment:
this value is wrong, the `calculated fee` results in 0.4 and in reality is over 12
![image](https://user-images.githubusercontent.com/93129175/228573338-1badf7fb-3514-49d3-97d9-08d44f6480f3.png)